Tests/Controller/HelperControllerTest.php

```diff
@@ -328,6 +329,20 @@ public function testappendFormFieldElementAction($validatorInterface)
 
         $twig = new \Twig_Environment($this->getMock('\Twig_LoaderInterface'));
         $twig->addExtension(new FormExtension($mockRenderer));
+
+        if (Kernel::MAJOR_VERSION >= 3 && Kernel::MINOR_VERSION >= 2) {
```

Comment:
Here is your answer right from the sf docs : https://symfony.com/doc/current/setup/bundles.html#updating-your-code-to-support-symfony-2-x-and-3-x-at-the-same-time